library/ssl_misc.h

```diff
@@ -876,6 +876,9 @@ struct mbedtls_ssl_handshake_params
     1  -- MBEDTLS_SSL_EARLY_DATA_ENABLED (for use early data)
     */
     int early_data;
+#if defined(MBEDTLS_SSL_SRV_C)
+    int skip_failed_decryption;
```

Comment:
It would be good to indicate (in the comments) what the allowed value for skip_failed_decryption are.